src/PrestaShopBundle/Form/Admin/Sell/Product/Stock/StockOptionsType.php

```diff
@@ -94,6 +94,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'default_empty_data' => 0,
                 'empty_view_data' => null,
                 'modify_all_shops' => true,
+                // @todo: need to trigger opening allShopscheckbox on "disabling_switch" change too.
```

Comment:
I still don't understand this bug exactly but I think it's related to the fact that `low_stock_threshold` is split into two fields in the DB, we can check this tomorrow through peer programming

Reply:
As we checked there is a workaround to make this work, as long as you click/focus on the associated field or change its value That's the initial behaviour of the multishop checkbox so I think it's fine for now
We can improve this later if we get feedback that the UX is not good enough

Reply:
Raised related issue based on our private discussion here https://github.com/PrestaShop/PrestaShop/issues/30750